src/lib/mina_stdlib/timeout_lib.ml

```diff
@@ -85,8 +85,6 @@ module Make (Time : Time_intf) : Timeout_intf(Time).S = struct
       Deferred.create (fun ivar ->
           ignore
             ( create time_controller timeout_duration ~f:(fun x ->
-                  if Ivar.is_full ivar then
```

Comment:
This is removed as it seems to be an old debugging statement. It does remove the dependency on logger for timeout_lib, and therefore for mina_stdlib, keeping the simplicity of mina_stdlib.

Reply:
Let's maybe print to stderr?
Otherwise I'd prefer to have some analysis of usages of this library and its code to ensure that this codepath is truly never activated.